docs/admin/user-management.rst

```diff
@@ -129,18 +138,16 @@ CrateDB clusters is also part of that list.
 To list all existing users query the table::
 
     cr> SELECT * FROM sys.users order by name;
-    +-------------+----------+-----------+
-    | name        | password | superuser |
-    +-------------+----------+-----------+
-    | Custom User |     NULL | FALSE     |
-    | crate       |     NULL | TRUE      |
-    | user_a      |     NULL | FALSE     |
-    | user_b      | ******** | FALSE     |
-    +-------------+----------+-----------+
+    +-------------+----------------------------------------------------------------------------------+----------+-----------+
+    | name        | parents                                                                          | password | superuser |
+    +-------------+----------------------------------------------------------------------------------+----------+-----------+
+    | Custom User | []                                                                               | NULL     | FALSE     |
+    | crate       | []                                                                               | NULL     | TRUE      |
+    | user_a      | [{"grantor": "crate", "role": "role_a"}, {"grantor": "crate", "role": "role_b"}] | NULL     | FALSE     |
+    | user_b      | []                                                                               | ******** | FALSE     |
+    +-------------+----------------------------------------------------------------------------------+----------+-----------+
     SELECT 4 rows in set (... sec)
 
-The column ``name`` shows the unique name of the user, the column ``superuser``
-shows whether the user has superuser privileges or not.
```

Comment:
Why we remove this?

Reply:
Because it's reduntant imho, we already link to the the sys tables where we describe the columns in detail, but happy to revert it, if you think we should have it here as well.

Reply:
It's fine to remove, I just wanted to confirm that this info is still actual.